sensing/ahrs_ekf.cpp

```diff
@@ -394,7 +393,7 @@ Ahrs_ekf::Ahrs_ekf(const Imu& imu, ahrs_t& ahrs, const Ahrs_ekf::conf_t config):
     ahrs_.internal_state = AHRS_INITIALISING;
 }
 
-bool Ahrs_ekf::update(void)
+bool Ahrs_ekf::predict_and_update(void)
```

Comment:
What is the reason for this change ?
There should not be any name-clash issue with Kalman::update(z), if that is what you wanted to avoid.